<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteEditor</class>
 <widget class="QMainWindow" name="SpriteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpriteEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SpriteCanvas" name="canvasWidget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>171</width>
      <height>391</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>169</width>
       <height>389</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="SpritePreview" name="previewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="previewSpeedController">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="copyGroupButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy Group</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="groupSelectCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Pixels</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpInstructionsLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>55</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixels copied! Click to choose an anchor point and then click where you wish to paste.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="bucketFillBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bucket Fill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpriteCanvas</class>
   <extends>QWidget</extends>
   <header>spritecanvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpritePreview</class>
   <extends>QWidget</extends>
   <header location="global">spritepreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
